velox/vector/tests/VectorMaker.cpp

```diff
@@ -220,6 +220,47 @@ ArrayVectorPtr VectorMaker::arrayOfRowVector(
       0);
 }
 
+ArrayVectorPtr VectorMaker::arrayVector(
+    const std::vector<vector_size_t>& offsets,
+    const VectorPtr& elementVector,
+    const std::vector<vector_size_t>& nulls) {
+  BufferPtr offsetsBuffer = allocateOffsets(offsets.size(), pool_);
+  BufferPtr sizesBuffer = allocateSizes(offsets.size(), pool_);
+  BufferPtr nullsBuffer = nullptr;
+  auto rawOffsets = offsetsBuffer->asMutable<vector_size_t>();
+  auto rawSizes = sizesBuffer->asMutable<vector_size_t>();
+
+  std::vector<vector_size_t> sizes;
```

Comment:
Since we already have sizes buffer allocated, we don't need to create another sizes vector.